lib/crypto-primitives/src/Cryptography/Format/PKCS7.hs

```diff
@@ -0,0 +1,41 @@
+module Cryptography.Format.PKCS7
```

Comment:
I would be advocate of `Crypthography.Padding.PKCS7` We can have other ways/standards of padding. And Padding better reflects what it is, it is less about presentation, but filling in. Formatting is orthogonal to this concept

Reply:
Good idea -- will change to `Cryptography.Padding.PKCS7`. Thanks for the suggestion!

Reply:
@paweljakubas I've updated this to match your suggestion in 60651241e8766a7a7e92da227a87526b031358a4.